frontend/src/views/CellGuide/components/CellGuideCard/style.ts

```diff
@@ -57,7 +54,7 @@ export const CellGuideView = styled.div<CellGuideViewProps>`
 
 export const CellGuideWrapper = styled.div<CellGuideViewProps>`
   margin: 0 auto 80px;
-  width: ${(props) =>
```

Comment:
this was causing unintended x-scrolling when the viewport was smaller than the width